nginx/nginx.go

```diff
@@ -307,21 +309,16 @@ func (lb *nginxLoadBalancer) createConfig(update controller.IngressUpdate) ([]by
 func createNginxEntries(update controller.IngressUpdate) []*nginxEntry {
 	sortedIngressEntries := update.SortedByName().Entries
 	indexedEntries := make(map[string]*nginxEntry)
+	hostPaths := make(map[string]map[string]struct{})
```

Comment:
okay i finally groked it, a map of host to map of paths to nothing as go doesn't have sets